### DISCUSSION TITLE
Welcome to RESTEasy Discussions!

### DISCUSSION THREAD

**Discussion by jamezp** [resteasy/resteasy#2913] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajit2548** [comment]


**Reply by jamezp** [reply]
Just an FYI it's usually best to start a new thread for questions like this :)

You're mixing RESTEasy and Jersey. They are both Jakarta REST implementations and you should likely only be using one. What contaienr are you running in?
